### ISSUE TITLE
false positive in at-rule-conditional-no-parentheses

### ISSUE THREAD

**Issue by evilebottnawi** [kristerkari/stylelint-scss#379]
Input:
```
@function strip-unit($number) {
  @if type-of($number) == "number" and not unitless($number) {
    @return $number / ($number * 0 + 1);
  }

  @return $number;
}
```

Output:
```
 2:3  ✖  Unexpected () used to surround statements for @-rules   scss/at-rule-conditional-no-parentheses
```

Should be no errors

**Comment by kristerkari** [comment]
Thanks!
